src/targets/php/guzzle.js

```diff
@@ -25,9 +25,11 @@ module.exports = function (source, options) {
   const code = new CodeBuilder(opts.indent);
 
   if (!opts.noTags) {
-    code.push(opts.shortTags ? '<?' : '<?php').blank();
+    code.push(opts.shortTags ? '<?' : '<?php');
   }
 
+  code.push("require_once('vendor/autoload.php');").blank();
```

Comment:
I mostly added this into the snippet because running `php path/to/snippet.php` in these integration tests was failing because the Composer autoloader wasn't getting picked up and there isn't an easy way to tell PHP where to look for it without dealing a custom `php.ini` file that I have zero interest in messing around with.